account_invoice_mgmt_dms/views/account_move_views.xml

```diff
@@ -83,7 +83,14 @@
                     string="Generate Document Manager"
                     type="object"
                     attrs="{'invisible': ['|', ('id', '=', False), '|',('ref', '=', False), '|',('state', '!=', 'posted'), ('state_complete_proceesing', '=', 'approved')]}"/>
-            </xpath>
+                <button 
+                    name="create_dms_file_document_manager"
+                    string="Generate Document Manager with validation"
+                    type="object"
+                    context ="{'validate': True}"
```

Comment:
Remove unneeded space between attribute name and value. And, also, context variable name should be more specific, as context is a very general variable container for Odoo:
```suggestion
                    context="{'create_dms_file_document_validate': True}"
```